src/SourceBuild/content/eng/bootstrap/buildBootstrapPreviouslySB.csproj

```diff
@@ -3,105 +3,38 @@
-  </ItemGroup>
-
   <!-- 
     These packages will be replaced with ms-built packages downloaded from official package feeds.
     The runtime packs do not have a RIDless package, so we'll use the version of their corresponding ref pack
     to determine the version.
   -->
   <ItemGroup>
-    <RuntimePack Include="Microsoft.AspNetCore.App.Runtime" Version="[$(MicrosoftAspNetCoreAppRefPackageVersion)]" />
-    <RuntimePack Include="Microsoft.NETCore.App.Crossgen2" Version="[$(MicrosoftNETCoreAppRefPackageVersion)]" />
-    <RuntimePack Include="Microsoft.NETCore.App.Host" Version="[$(MicrosoftNETCoreAppRefPackageVersion)]" />
-    <RuntimePack Include="Microsoft.NETCore.App.Runtime" Version="[$(MicrosoftNETCoreAppRefPackageVersion)]" />
-
-    <PortablePackage Include="Microsoft.DotNet.ILCompiler" Version="[$(MicrosoftDotNetILCompilerVersion)]" />
-    <PortablePackage Include="Microsoft.NETCore.DotNetAppHost" Version="[$(MicrosoftNETCoreDotNetAppHostVersion)]" />
-    <PortablePackage Include="Microsoft.NETCore.ILAsm" Version="[$(MicrosoftNETCoreILAsmVersion)]" />
-    <PortablePackage Include="Microsoft.NETCore.ILDAsm" Version="[$(MicrosoftNETCoreILDAsmVersion)]" />
-    <PortablePackage Include="Microsoft.NETCore.TestHost" Version="[$(MicrosoftNETCoreTestHostVersion)]" />
-
-    <PortablePackage Include="System.IO.Ports" Version="[$(RuntimeNativeSystemIOPortsVersion)]" IsNative="true" />
+    <PackageDownload Include="Microsoft.AspNetCore.App.Runtime.$(PortableRid)" Version="[$(MicrosoftAspNetCoreAppRefPackageVersion)]" />
+    <PackageDownload Include="Microsoft.NETCore.App.Crossgen2.$(PortableRid)" Version="[$(MicrosoftNETCoreAppRefPackageVersion)]" />
+    <PackageDownload Include="Microsoft.NETCore.App.Host.$(PortableRid)" Version="[$(MicrosoftNETCoreAppRefPackageVersion)]" />
+    <PackageDownload Include="Microsoft.NETCore.App.Runtime.$(PortableRid)" Version="[$(MicrosoftNETCoreAppRefPackageVersion)]" />
+
+    <PackageDownload Include="runtime.$(PortableRid).Microsoft.DotNet.ILCompiler" Version="[$(MicrosoftDotNetILCompilerVersion)]" />
+    <PackageDownload Include="runtime.$(PortableRid).Microsoft.NETCore.DotNetAppHost" Version="[$(MicrosoftNETCoreDotNetAppHostVersion)]" />
+    <PackageDownload Include="runtime.$(PortableRid).Microsoft.NETCore.ILAsm" Version="[$(MicrosoftNETCoreILAsmVersion)]" />
+    <PackageDownload Include="runtime.$(PortableRid).Microsoft.NETCore.ILDAsm" Version="[$(MicrosoftNETCoreILDAsmVersion)]" />
+    <PackageDownload Include="runtime.$(PortableRid).Microsoft.NETCore.TestHost" Version="[$(MicrosoftNETCoreTestHostVersion)]" />
+
+    <PackageDownload Include="runtime.$(PortableRid).runtime.native.System.IO.Ports" Version="[$(RuntimeNativeSystemIOPortsVersion)]" />
     <!-- These packages don't actually exist -->
     <ExcludedPackage Include="runtime.linux-musl-x64.runtime.native.System.IO.Ports" />
     <ExcludedPackage Include="runtime.linux-musl-arm64.runtime.native.System.IO.Ports" />
```

Comment:
This should be deleted. Apparently, these packages do exist now starting in 9.0.

Reply:
Removed in https://github.com/dotnet/sdk/pull/46575/commits/b3fd31394d41155ade89f341d98e0ab94b817928